content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -370,6 +371,29 @@ fn despawn_players(mut players: Query<(Entity, PlayerQuery)>) {
 }
 ```
 
+## Documentation improvements
+
+<div class="release-feature-authors">authors: @alice-i-cecile and many more</div>
+
+Great docs make learning, using and building Bevy better.
+But as a young engine, they're still a work-in-progress.
+
+Our docs team (led by `@alice-i-cecile`) has started to [systematically fix that](https://github.com/bevyengine/bevy/issues/3492), with the help of Rust's `#[warn(missing_docs)]` lint.
+Since 0.6, we've fully documented (and prevented doc-regressions for):
+
+* `bevy_tasks` by `@james7132`
+* `bevy_app` by `@dbearden`
+* `bevy_dylib` by `@KDecay`
+* `bevy_internal` by `@sheepyhead`
+
+There have been a huge number of [other doc improvements](https://github.com/bevyengine/bevy/pulls?q=is%3Apr+is%3Aclosed+label%3AC-Docs) over this time period as well, including the addition of many helpful [doc tests](https://doc.rust-lang.org/rustdoc/documentation-tests.html).
+A huge thanks to all of the contributors and reviewers who have helped make Bevy's docs better.
+
+For many people, the best way to learn a tool is to see it in action.
+We've been steadily polishing our [examples](https://github.com/bevyengine/bevy/tree/latest/examples):
+better explanations, more coverage, and higher code quality.
+If you're new to Bevy, check out the much-improved [Breakout example](https://github.com/bevyengine/bevy/blob/latest/examples/games/breakout.rs)!
```

Comment:
It would have been nice to have my sound PR merged, but it's not a priority compared to releasing 0.7